src/event/base.rs

```diff
@@ -158,15 +163,19 @@ impl EventBase {
         }
     }
 
-    pub fn event_add(&self, event: &EventHandle, timeout: Duration) -> c_int {
-        let tv = to_timeval(timeout);
+    pub fn event_add(&self, event: &EventHandle, timeout: Option<Duration>) -> c_int {
```

Comment:
Would you mind also updating the example to reflect this new signature? Eventually I'll have CI set up to do the chirping..

https://github.com/jmagnuson/libevent-rs/blob/57a60f9c3a56f80d0cf244bc745ad33843d8e4eb/examples/hello/src/main.rs#L36

I think that's the only change required.